src/Package/Impl/ExportDialog/ExportImageDialog.xaml.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Windows;
+using System.Windows.Controls;
+using System.ComponentModel;
+using Microsoft.Common.Core;
+using Microsoft.VisualStudio.R.Package.Wpf;
+
+namespace Microsoft.VisualStudio.R.Package.ExportDialog {
+    public partial class ExportImageDialog : PlatformDialogWindow, INotifyPropertyChanged {
+
+        private int _userWidth;
+        private int _userHeight;
+        private bool _viewPlot;
+        private const int MIN = 100;
+        private const int MAX = 100000;
+        private int _prevValidWidth;
+        private int _prevValidHeight;
+        private bool _maintainAspectRatio;
+        private int _originalWidth;
+        private int _originalHeight;
+
+        public ExportImageDialog(ExportArguments imageArguments) {
+            InitializeComponent();
+            this.HasMaximizeButton = false;
+            this.HasMinimizeButton = false;
+            UserWidth = imageArguments.PixelWidth;
+            UserHeight = imageArguments.PixelHeight;
+            this.DataContext = this;
+        }
+
+        public int UserWidth {
```

Comment:
This is clearly a ViewModel properties which should be moved into a separate object (see any existing `*ViewModel` class in `Microsoft.R.Components` for the reference).